ramls/_schemas/kv_configuration.schema

```diff
@@ -30,6 +30,15 @@
     "userId": {
       "type": "string"
     },
+    "recordsSource": {
+      "type": "string",
+      "default": "Source record storage",
```

Comment:
How does this default make sense for configuration records that are defined by modules that have nothing to do with source records / inventory?